css/styles.css

```diff
@@ -0,0 +1,566 @@
+* {
+  margin: 0;
+  padding: 0;
+  color: var(--lightBlack);
+  font-family: 'Lato', sans-serif;
+  scroll-behavior: smooth;
+}
+
+.nav-social,
+.nav-bar,
+.mi-hidden,
+.partners,
+.footer {
+  display: none;
+}
+
+.nav,
```

Comment:
- [ ] As we are imitating the Behance design provided for this milestone of the project, your project should look like the one as the design chosen and if we compare your design and the one we are imitating are not really the same on some point I think you should improve on this so that can attract recruiters :wink: 
Your Design :point_down: 
![top34](https://user-images.githubusercontent.com/88894525/203973867-2d315f02-b9f2-4cae-bfe8-48dfcc2b33c1.png)


The design provided 
![Screenshot from 2022-11-25 13-02-41](https://user-images.githubusercontent.com/88894525/203973678-435d59f6-42eb-4b00-8e25-2d6a458183e2.png)